### DISCUSSION TITLE
problem when deleting a report loaded incorrectly with an extension other than .py in netbox 3.5.7

### DISCUSSION THREAD

**Discussion by Csr22222** [netbox-community/netbox#16196] [Help Wanted!] [answered]
Good morning, by mistake I uploaded a file to netbox in the reports section (a file with a .pdf extension), which now causes the reports page not to load, if I just delete the .pdf file from the reports folder, it continues sending a error because in some configuration file or in the database a reference is made to that .pdf file. can someone help me with this problem

**Comment by markkuleinio** [comment]
Did you already try to restart NetBox services (`netbox` and `netbox-rq`)? A long shot but I remember some report/script issues solved by service restart.

**Reply by Csr22222** [reply]
I delete the pdf file from reports folder & restart the netbox service, and the page still does not load and tells me that the deleted file is needed, so what I want to do is delete the entry that refers to that file and see if the error is corrected, but I don't know where or how to do it.

**Reply by candlerb** [reply, marked as answer]
You could create a zero-byte file with the same name (`touch /path/to/file.pdf`)

**Reply by Csr22222** [reply]
Placing the file with the same name but empty, it did allow me to access the reports section and be able to delete them. thank you very much, greetings.
